src/pages/cli/geo/geofencing.mdx

```diff
@@ -0,0 +1,283 @@
+
+If you haven't set up the `auth` category already, you can do so by running:
+```bash
+amplify add auth
+```
+
+When you have `auth` category set up, you can add the Cognito User Pool groups using:
+```bash
+amplify update auth
+```
+```console
+? What do you want to do? 
+  Apply default configuration with Social Provider (Federation) 
+  Walkthrough all the auth configurations 
+❯ Create or update Cognito user pool groups 
+  Create or update Admin queries API 
+
+? Provide a name for your user pool group: storesInWashingtonGeofenceCollectionAdmin
+
+? Do you want to add another User Pool Group No
+
+? Sort the user pool groups in order of preference
+  storesInWashingtonGeofenceCollectionAdmin
+
+```
+
+When you have at least one Cognito User Pool group added, you can now provision a new Geofence Collection using:
+```bash
+amplify add geo
+```
+```console
+? Select which capability you want to add: 
+  Map (visualize the geospatial data) 
+  Location search (search by places, addresses, coordinates) 
+❯ Geofencing (visualize virtual perimeters)
+```
+
+Next, set a name for the Geofence Collection: 
+
+```console
+? Provide a name for the Geofence Collection: storesInWashington
+```
+
+## Geofence Collection Access permissions
+
+Next, configure the access permissions for your geofence collection resource and authorize users of desired Cognito User Pool groups to perform `CRUD` operations on Geofences in the collection.
+
+```console
+? Select one or more cognito groups to give access:
+✔ storesInWashingtonGeofenceCollectionAdmin
+
+? What kind of access do you want for storesInWashingtonGeofenceCollectionAdmin users? Select ALL that apply:
+ ✔ Read geofence
+ ✔ Create/Update geofence
+   Delete geofence
+ ✔ List geofences
+```
+
+The above chosen permissions are granted to the users belonging the specified Cognito User Pool group. 
+Please note that these permissions apply to ALL Geofences in a collection. For example, If you chose `Read geofence` permission for say `storesInWashingtonGeofenceCollectionAdmin`
+Cognito group, ALL users added to that group will be able to read the properties of ALL Geofences in that Geofence collection.
+
+## Geofence Collection Pricing Plan and Data Provider
+
+Amazon Location Service offers three pricing plans namely, `RequestBasedUsage`, `MobileAssetTracking` and `MobileAssetManagement` that require no up-front commitment, and no minimum fee.
+You select one of the following pricing plans for the Amazon Location Service resources you create for your application.
+Amplify helps you determine the best pricing plan for you by guiding you through a set of questions.
+These pricing plan related set of questions are presented to you when you add the first Geofence Collection resource to your application and can be updated later if needed. 
+Once you chose the pricing plan, it will be automatically used for the ALL Geo resources added to your application.
```

Comment:
Should we look into removing this

Reply:
I will have another PR once the tag is updated with pricing plan removal changes